buildpack.toml

```diff
@@ -0,0 +1,306 @@
+      type = "Apache-2.0"
+      uri = "https://www.apache.org/licenses/"
+
+  [[metadata.dependencies]]
+    cpes = ["cpe:2.3:a:apache:tomee-plume:9.0.0-M7:*:*:*:*:*:*:*"]
+    id = "tomee-plume"
+    name = "Apache Tomee - Plume"
+    purl = "pkg:generic/apache-tomee@9.0.0-M7"
+    sha256 = "fa13338cddea7c4c0061fbb03f87c6deba93187d4da176eb68e812657d279a68"
+    stacks = ["io.buildpacks.stacks.bionic", "io.paketo.stacks.tiny", "*"]
+    uri = "https://downloads.apache.org/tomee/tomee-9.0.0-M7/apache-tomee-9.0.0-M7-plume.tar.gz"
+    version = "9.0.0-M7"
+
+    [[metadata.dependencies.licenses]]
+      type = "Apache-2.0"
+      uri = "https://www.apache.org/licenses/"
+
+  [[metadata.dependencies]]
+    cpes = ["cpe:2.3:a:cloudfoundry:tomcat-access-logging-support:3.3.0:*:*:*:*:*:*:*"]
+    id = "tomcat-access-logging-support"
+    name = "Apache Tomcat Access Logging Support"
+    purl = "pkg:generic/tomcat-access-logging-support@3.3.0"
+    sha256 = "49f9acac1bed65c683276d662c985ed63c685a603e7f5b72f732673bb826a0d4"
+    stacks = ["io.buildpacks.stacks.bionic", "io.paketo.stacks.tiny", "*"]
+    uri = "https://repo.spring.io/release/org/cloudfoundry/tomcat-access-logging-support/3.3.0.RELEASE/tomcat-access-logging-support-3.3.0.RELEASE.jar"
+    version = "3.3.0"
+
+    [[metadata.dependencies.licenses]]
+      type = "Apache-2.0"
+      uri = "https://github.com/cloudfoundry/java-buildpack-support/blob/main/LICENSE"
+
+  [[metadata.dependencies]]
+    cpes = ["cpe:2.3:a:cloudfoundry:tomcat-lifecycle-support:3.3.0:*:*:*:*:*:*:*"]
+    id = "tomcat-lifecycle-support"
+    name = "Apache Tomcat Lifecycle Support"
+    purl = "pkg:generic/tomcat-lifecycle-support@3.3.0"
+    sha256 = "b725f4cae0be4a999a08ed978b29081edfd68a187a926ed740cc1f8e9aca70ce"
+    stacks = ["io.buildpacks.stacks.bionic", "io.paketo.stacks.tiny", "*"]
+    uri = "https://repo.spring.io/release/org/cloudfoundry/tomcat-lifecycle-support/3.3.0.RELEASE/tomcat-lifecycle-support-3.3.0.RELEASE.jar"
+    version = "3.3.0"
+
+    [[metadata.dependencies.licenses]]
+      type = "Apache-2.0"
+      uri = "https://github.com/cloudfoundry/java-buildpack-support/blob/main/LICENSE"
+
+  [[metadata.dependencies]]
+    cpes = ["cpe:2.3:a:cloudfoundry:tomcat-logging-support:3.3.0:*:*:*:*:*:*:*"]
+    id = "tomcat-logging-support"
+    name = "Apache Tomcat Logging Support"
+    purl = "pkg:generic/tomcat-logging-support@3.3.0"
+    sha256 = "46db58c1d8ef2555d7053a12d669daefb676e9a17a2a1771cd60c95bcb210b0d"
+    stacks = ["io.buildpacks.stacks.bionic", "io.paketo.stacks.tiny", "*"]
+    uri = "https://repo.spring.io/release/org/cloudfoundry/tomcat-logging-support/3.3.0.RELEASE/tomcat-logging-support-3.3.0.RELEASE.jar"
+    version = "3.3.0"
+
+    [[metadata.dependencies.licenses]]
+      type = "Apache-2.0"
+      uri = "https://github.com/cloudfoundry/java-buildpack-support/blob/main/LICENSE"
+
+[[stacks]]
+  id = "io.buildpacks.stacks.bionic"
+
+[[stacks]]
+  id = "io.paketo.stacks.tiny"
```

Comment:
This is a new buildpack, so I don't think we need these. The `"*"` matches everything. I think we just had these in for historical reasons in our existing buildpacks.

Reply:
done